physionet-django/console/templates/console/training_type/index.html

```diff
@@ -0,0 +1,74 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}Training Types{% endblock %}
+
+{% block content %}
+ <div class="card mb-3">
+  <div class="card-header">
+    Training Types <span class="badge badge-pill badge-info">{{ training_types|length }}</span>
+  </div>
+  <div class="card-body">
+    <div><button type="button" class="btn btn-sm btn-danger" data-toggle="modal" data-target="#op-training">Create/Update On Platform Training</button></div>
+    <div class="table-responsive">
+      <table class="table table-bordered">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Valid Duration</th>
+            <th>On Platform Training</th>
+          </tr>
+        </thead>
+        <tbody>
+        {% for training in training_types %}
+            <tr>
+                <td>{{ training.name|title }}</td>
+                <td>{{ training.valid_duration }}</td>
+                <td>{% if training.required_field == 2 %} True {% else %} False {% endif %}</td>
+            </tr>
+        {% endfor %}
+        </tbody>
+      </table>
+    </div>
+
+    <div class="modal fade" id="op-training" tabindex="-1">
+        <div class="modal-dialog">
+            <form action="{% url 'op_training' %}" method="POST" enctype="multipart/form-data" class="">
+                <div class="modal-content">
+                    <div class="modal-body">
+                        <p>Download Sample Json File by <a href="{% static 'sample/create.json' %}" download>Clicking Here</a></p>
+                        <div>
+                            {% csrf_token %}
+                            <div class="form-group">
+                                <label>Training Type: </label>
+                                <select name="training_id" class="form-control" required>
+                                    <option disabled>Choose...</option>
+                                    <option value="-1">Create new training</option>
```

Comment:
In places like this, I think "course" is clearer than "training". e.g. "Create new course" or "Create new training course". There's something about "training" being uncountable (and countable?) that can make it a confusing word to use.